README.md

```diff
@@ -69,32 +70,41 @@ struct ErrorModel: ErrorModeling {
     }
 }
 
-struct LoadedModel: LoadedModeling {
+struct BlogModel: BlogModeling {
     var title: String
     var body: String
+    func refresh() -> AnyPublisher<BlogArticleViewState, Never> {
+        ...
+    }
 }
 ```
 
 ### View Definition
 
-The view observes and renders the state using the `StateContainer` type. State changes will automatically update the view.
+The view observes and renders the state using the `ViewState` property wrapper. State changes will automatically update the view.
 
 ```swift
-struct BlogEntryView: View, ViewStateRendering {
-    @StateObject var container: StateContainer<BlogEntryViewState>
+struct BlogEntryView: View {
+    @ViewState var state: BlogEntryViewState
 
     init() {
-        _container = .init(state: .initialized(LoaderModel()))
+        _state = .init(wrappedValue: .initialized(LoaderModel()))
```

Comment:
I don't want your head to explode, and I am sure you thought about how to do this if it was even possible, but... Wouldn't it be nice if this line could instead read
```swift
state = .initialized(LoaderModel())
```
😄 
Oh the ergonomics! So nice, if it were possible....

Reply:
This is technically possible, but I'd have to make the `ViewState` wrapped value become mutable, which means engineers could update the current state directly instead of requiring them to go through the `observe(...)` functions of the `StateContainer`. I'm not 100% sure of the ramifications of such a change. Maybe you could help me think through them...

The ViewState property wrapper implementation would have to change change to:
```swift
    public var wrappedValue: State {
        get { container.state }
        set { container.observe(newValue) }
    }
// instead of
    public var wrappedValue: State {
        get { container.state }
    }
```

That change would allow this directly in the view:
```swift
Button("Go") {
    state = .gone
}
```

It could be argued that it is no different (an no more of a concession) than what's already possible:
```swift
Button("Go") {
    $state.observe(.gone)
}
```

Reply:
Making `ViewState` mutable is probably the wrong choice, because that lifts an important guardrail and puts at risk the entire reason for using VSM: Predictabilty. It won't do to allow the possibility that state could be changed from anywhere.

In the two Button examples, you have to admit the first one (with `state = .gone`) is super easy to reason about. But in the second, I actually appreciate the extra context about what is really happening, which ironically is only useful if the reader really understands what it means to observe the state with a future state value (the state achieved by tapping the button.)

Always tradeoffs. But I think the mutability issue is the deal breaker. Thanks for playing. 😄

Reply:
In this particular particular situation, since the `LoaderModel` initializer has no parameters, you could get rid of the initializer entirely and just set this at property definition:
```swift
@ViewState var state: BlogEntryViewState = .initialized(LoaderModel())
```